pr_testing/test_multiple_prs.sh

```diff
@@ -79,19 +79,34 @@ function process_changed_files() {
   sort -u "$directlyChangedFiles" $WORKSPACE/indirectly-changed-files.txt > "$allChangedFiles"
 }
 
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"  # Absolute path to script
+CMS_BOT_DIR=$(dirname ${SCRIPTPATH})  # To get CMS_BOT dir path
+
+readarray -t ALL_GPU_TYPES < ${CMS_BOT_DIR}/gpu_flavors.txt
+
+declare -a ENABLE_GPU_FLAVORS
+for ex_type in ${EXTRA_RELVALS_TESTS} ; do
+  [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep "^${ex_type}$" | wc -l) -gt 0 ] || continue
```

Comment:
@iarspider , please remove this line and just loop over `ENABLE_BOT_TESTS`

Reply:
Done